wcivf/apps/elections/templates/elections/elections_view.html

```diff
@@ -128,7 +140,20 @@ <h2>Past Elections</h2>
             }
         });
 
-
+        function checkHideDate() {
+            for (var i = 0; i < date_grouper.length; i++) {
+                all_hidden = true;
+                date_grouper_urls = date_grouper[i].querySelectorAll('li a');
+                for (var j = 0; j < date_grouper_urls.length; j++) {
+                    if (date_grouper_urls[j].style.display == '' && date_grouper_urls[j] in local) {
```

Comment:
Stuck. When I click on a filter button (ie "local") and there are no local elections on a given date, I want to hide that date. And how to do this dynamically based on the filter that is clicked. 🤔

Reply:
Hmm not sure if this is possible, but could you loop through every H3 with the `grouper-date` class, and look for the next `election-urls` div and check if there is a `<li>` element with the display set, and if there are not any, set the display for that `h3` element to none? Not sure how difficult that conditional is to do in JS but feels like something along those lines might work